lms/djangoapps/sneakpeek_deeplink/middleware.py

```diff
@@ -19,6 +22,8 @@
     'info',
 )
 
+log = logging.getLogger("edx.sneakpeek_middleware")
```

Comment:
@stvstnfrd So is anything outside of a method or class definition considered a constant?  I'm not an advocate of disabling either but I didn't think of log as a "constant".